src/collective/glossary/browser/static/resources.pt

```diff
@@ -0,0 +1,5 @@
+<link rel="stylesheet" tal:attributes="href string:${view/site_url}/++resource++collective.glossary/main.css" />
```

Comment:
this file can't be hardcoded, it needs to be generated by webpack also; check:

https://github.com/collective/collective.lazysizes/tree/4.1.4/webpack/app

Reply:
and I don't get the point of `main.css` file.. this should be incorporated into webpack generated file.

Reply:
This is historical: the tooltip css code was only shown for "context/@@glossary_state".

I suppose that all the CSS can be merged into one file.

Reply:
Merged all css in one file